regression-tests.dnsdist/test_Spoofing.py

```diff
@@ -913,3 +913,59 @@ def testLuaSpoofBasedOnStatistics(self):
             (_, receivedResponse) = sender(query, response=None, useQueue=False)
             self.assertTrue(receivedResponse)
             self.assertEqual(expectedResponseAfterwards, receivedResponse)
+
+class TestSpoofingLuaSpoofPacket(DNSDistTest):
+
+    _config_template = """
+    local ffi = require("ffi")
+
+    function spoofpacket(dq)
+        -- REFUSED answer
+        local rawResponse="\\000\\000\\129\\133\\000\\001\\000\\000\\000\\000\\000\\000\\014lua\\045raw\\045packet\\012ffi\\045spoofing\\005tests\\008powerdns\\003com\\000\\000\\001\\000\\001"
+        local qtype = ffi.C.dnsdist_ffi_dnsquestion_get_qtype(dq)
```

Comment:
Perhaps check that the `qtype` is `DNSQType.A` and do nothing otherwise? While it does not matter for that test, I'm always fearful someone decides to copy/paste the code of our tests into production..